vllm/entrypoints/openai/serving_engine.py

```diff
@@ -443,29 +443,30 @@ async def _preprocess_chat(
             tokenizer,
         )
 
+        _chat_template_kwargs = dict(
+            add_generation_prompt=add_generation_prompt,
+            continue_final_message=continue_final_message,
+            tools=tool_dicts,
+            documents=documents,
+        )
+        _chat_template_kwargs.update(chat_template_kwargs or {})
+
         request_prompt: Union[str, List[int]]
         is_mistral_tokenizer = isinstance(tokenizer, MistralTokenizer)
         if is_mistral_tokenizer:
             request_prompt = apply_mistral_chat_template(
                 tokenizer,
                 messages=messages,
                 chat_template=chat_template,
-                add_generation_prompt=add_generation_prompt,
-                continue_final_message=continue_final_message,
-                tools=tool_dicts,
-                documents=documents,
-                **(chat_template_kwargs or {}),
+                **_chat_template_kwargs,
             )
         else:
             request_prompt = apply_hf_chat_template(
                 tokenizer,
                 conversation=conversation,
                 chat_template=chat_template,
-                add_generation_prompt=add_generation_prompt,
-                continue_final_message=continue_final_message,
-                tools=tool_dicts,
-                documents=documents,
-                **(chat_template_kwargs or {}),
+                tokenize=False,
```

Comment:
Is this intended? This argument wasn't there before.

Reply:
Yes, for pass mypy type checking. When using `**dict` in function, you need to explicitly call all the typed parameters defined by the function, otherwise it will not pass the mypy type check.

Reply:
Can you add a note here to indicate this?